app/models/miq_worker/runner.rb

```diff
@@ -90,29 +90,6 @@ def worker_monitor_drb
     end
   end
 
-  ###############################
-  # VimBrokerWorker Methods
-  ###############################
-
-  def self.delay_startup_for_vim_broker?
-    !!@delay_startup_for_vim_broker
-  end
-
-  class << self
-    attr_writer :delay_startup_for_vim_broker
-  end
-
-  def self.delay_queue_delivery_for_vim_broker?
-    !!@delay_queue_delivery_for_vim_broker
-  end
-
-  class << self
-    attr_writer :delay_queue_delivery_for_vim_broker
-
-    alias require_vim_broker? delay_queue_delivery_for_vim_broker?
```

Comment:
There are a couple of callers for this in manageiq-providers-vmware

```
/Users/jfrey/dev/manageiq-providers-vmware/app/models/manageiq/providers/vmware/infra_manager/refresh_worker/runner.rb:3:  self.require_vim_broker           = !Settings.ems_refresh.vmwarews.streaming_refresh
/Users/jfrey/dev/manageiq-providers-vmware/app/models/manageiq/providers/vmware/infra_manager/metrics_collector_worker/runner.rb:2:  self.require_vim_broker = true
```

Reply:
Yeah sorry I didn't link that this had a dependent PR, these are removed in https://github.com/ManageIQ/manageiq-providers-vmware/pull/506